tests/integration/cgroups.bats

```diff
@@ -119,6 +119,86 @@ function setup() {
 	#
 }
 
+@test "runc exec (cgroup v2, ro cgroupfs, new cgroupns) does not chown cgroup" {
+	requires root cgroups_v2 systemd
+
+	set_cgroups_path
+
+	# configure a user namespace
+	update_config '.linux.namespaces += [{"type": "user"}]'
+	update_config '.linux.uidMappings += [{"hostID": 100000, "containerID": 0, "size": 65536}]'
+	update_config '.linux.gidMappings += [{"hostID": 100000, "containerID": 0, "size": 65536}]'
+
+	# chown test temp dir to allow host user to read it
+	chown 100000 "$ROOT"
+
+	# chown rootfs to allow host user to mkdir mount points
+	chown 100000 "$ROOT"/bundle/rootfs
```

Comment:
As you're doing this chowns and update_config in every test, maybe it make sense to move these to a separate file (e.g. `cgroup_delegate.bats`) and move the common part to `setup()` function.

Reply:
Moving the tests to a separate file, sure.

Extracting the config manipulation to `setup()`, sure.

Unfortunately, these `chown`s can't be moved, because they need to be guarded by `requires root`.  Maybe that could also be moved to `setup()` but that feels wrong and likely to confuse.

Reply:
But I did extract those `chown`s to a helper subroutine.